lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/util/KBVAnswerStorageMapper.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.di.ipv.cri.kbv.api.util;
+
+import uk.gov.di.ipv.cri.kbv.api.domain.KBVAnswerItem;
+import uk.gov.di.ipv.cri.kbv.api.domain.KbvQuestion;
+import uk.gov.di.ipv.cri.kbv.api.domain.QuestionsResponse;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class KBVAnswerStorageMapper {
+    public List<KBVAnswerItem> mapToKBVAnswerItems(QuestionsResponse questionsResponse) {
+        List<KBVAnswerItem> kbvAnswerItems = new ArrayList<>();
+        for (KbvQuestion kbvQuestion : questionsResponse.getQuestions()) {
```

Comment:
given that you are using `map()` in the method below you may as well use it here too. 
e.g. `Arrays.stream(questionResponse.getQuestions()).map(`
Then you can simply return the result of the mapping invocation & no need to declare any variables & use a `for` loop

It would also make sense to add a `filter` here that does the check for 
Objects.nonNull(kbvQuestion.getQuestionOptions())
                && Objects.nonNull(kbvQuestion.getQuestionOptions().getOptions()
and check that the length of getOptions is more than zero                

This would mean that the `getAnswersForQuestion` method is only invoked for questions with a non-empty collection of options